remotedb/remotedb.go

```diff
@@ -218,5 +220,43 @@ func (itr *iterator) Value() []byte {
 }
 
 func (itr *iterator) Close() {
-	// TODO: Shut down the iterator
+	err := itr.dic.CloseSend()
```

Comment:
weird method name... does `CloseSend` mean close for sending?

Reply:
Close the send-side, I think. Modeled after the example at the bottom of https://grpc.io/docs/tutorials/basic/go.html.